### PR TITLE
Create export microservice - Closes #861

### LINKED ISSUE #861: Create core export microservice
### Description

Create a new microservice based on Lisk Service Template.

Name it `lisk-service-export`.

### Motivation

- The microservice is needed for all core-related export features

### Acceptance Criteria

- New service is created
- Recent version of lisk-service-framework is used
- The service starts and connects to default message broker

### PR DESCRIPTION
**Pull request by MichalTuleja**
### What was the problem?

This PR resolves #861

### How was it solved?

- [x] Added microservice based on template
- [x] Updated Docker-compose file
- [x] Updated PM2 config
- [x] Updated README
- [x] Updated Jenkinsfile
- [x] Bumped Node.js to 14.x

### How was it tested?

Jenkins

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ Each service is an independent part of the repository and is placed in a separat
 | [Lisk](services/core) | The Lisk Core service acts as a bridge between the Lisk Core and the Lisk Service API. Its main purpose is to provide enriched data from the Lisk Core API. This service is aimed at providing high availability, and both efficient and reliable access to the Lisk Core API. |
 | [Market](services/market) | The Market service allows price data retrieval. It supports multiple sources to keep the current Lisk token price up-to-date and available to the clients in real time. |
 | [Newsfeed](services/newsfeed) | The Newsfeed service is a single-purpose microservice that aggregates content sharing platforms and shares collected data with UI clients such as Lisk Desktop. |
+| [Export](services/template) | TBD |
```
Comment (sameersubudhi) [inline README.md:25]:
```suggestion
| [Export](services/export) | TBD |
```

File: Makefile [hunk 2]
```diff
@@ -66,6 +66,9 @@ build-gateway:
 	cd ./services/gateway && docker build --tag=lisk/service_gateway ./
 
 build-template:
+	cd ./services/export && docker build --tag=lisk/service_export ./
+
+build-export:
 	cd ./services/template && docker build --tag=lisk/service_template ./
```
Comment (sameersubudhi) [inline Makefile:72]: Please cherry-pick the commit `4efb5d4b4b812c747fad6006b146d00cf358f4a4` to fix the commands (build-export currently builds template and vice-versa)

File: docker-compose.yml [hunk 3]
```diff
@@ -177,15 +177,44 @@ services:
...
+      - services_network
+      - export_network
+    environment:
+      - SERVICE_BROKER=redis://redis_common:6379/0
+      - SERVICE_MARKET_REDIS=redis://redis_market:6379/0
+      - SERVICE_MARKET_FIAT_CURRENCIES=${SERVICE_MARKET_FIAT_CURRENCIES}
+      - SERVICE_MARKET_TARGET_PAIRS=${SERVICE_MARKET_TARGET_PAIRS}
+    restart: always
+
+  export_market:
```
Comment (sameersubudhi) [inline docker-compose.yml:195]:
```suggestion
  redis_export:
```

File: services/export/app.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2019 Lisk Foundation
```
Comment (sameersubudhi) [inline services/export/app.js:3]:
```suggestion
 * Copyright © 2021 Lisk Foundation
```

File: services/export/app.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+	version: packageJson.version,
+};
+
+LoggerConfig(loggerConf);
+
+const logger = Logger();
+
+// Initialize Microservice framework
+const app = Microservice({
+	name: 'template',
```
Comment (sameersubudhi) [inline services/export/app.js:39]:
```suggestion
	name: 'export',
```

File: services/export/config.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2019 Lisk Foundation
```
Comment (sameersubudhi) [inline services/export/config.js:3]:
```suggestion
 * Copyright © 2021 Lisk Foundation
```

File: ecosystem.core3.config.js [hunk 7]
```diff
@@ -113,5 +113,22 @@ module.exports = {
...
+			out_file: './logs/service_export.log',
+			error_file: './logs/service_export.err',
+			log_date_format: 'YYYY-MM-DD HH:mm:ss SSS',
+			watch: false,
+			kill_timeout: 10000,
+			max_memory_restart: '512M',
+			autorestart: true,
+			env: {
+				SERVICE_BROKER: 'redis://localhost:6379/0',
+				SERVICE_EXPORT_REDIS: 'redis://localhost:6379/3',
```
Comment (sameersubudhi) [inline ecosystem.core3.config.js:130]: The environment variable is currently unused.

File: services/export/events/hello.js [hunk 8]
```diff
@@ -0,0 +1,29 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2019 Lisk Foundation
```
Comment (sameersubudhi) [inline services/export/events/hello.js:3]:
```suggestion
 * Copyright © 2021 Lisk Foundation
```

File: services/export/jobs/job.js [hunk 9]
```diff
@@ -0,0 +1,28 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2019 Lisk Foundation
```
Comment (sameersubudhi) [inline services/export/jobs/job.js:3]:
```suggestion
 * Copyright © 2021 Lisk Foundation
```

File: services/export/methods/hello.js [hunk 10]
```diff
@@ -0,0 +1,73 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2019 Lisk Foundation
```
Comment (sameersubudhi) [inline services/export/methods/hello.js:3]:
```suggestion
 * Copyright © 2021 Lisk Foundation
```

File: services/export/methods/status.js [hunk 11]
```diff
@@ -0,0 +1,29 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2019 Lisk Foundation
```
Comment (sameersubudhi) [inline services/export/methods/status.js:3]:
```suggestion
 * Copyright © 2021 Lisk Foundation
```

File: Makefile (hunk 2 above)
Comment (MichalTuleja) [reply Makefile]: Done

File: docker-compose.yml [hunk 12]
```diff
@@ -177,15 +177,44 @@ services:
...
+    image: lisk/service_export
+    depends_on:
+      - redis_common
+      - redis_export
+    networks:
+      - services_network
+      - export_network
+    environment:
+      - SERVICE_BROKER=redis://redis_common:6379/0
+      - SERVICE_MARKET_REDIS=redis://redis_market:6379/0
```
Comment (nagdahimanshu) [inline docker-compose.yml:190]: This should be:
```suggestion
      - SERVICE_EXPORT_REDIS=redis://redis_export:6379/0
```

File: docker-compose.yml [hunk 13]
```diff
@@ -177,15 +177,44 @@ services:
...
+      - SERVICE_MARKET_FIAT_CURRENCIES=${SERVICE_MARKET_FIAT_CURRENCIES}
```
Comment (nagdahimanshu) [inline docker-compose.yml:191]: This lines are not needed here:
 - SERVICE_MARKET_TARGET_PAIRS=${SERVICE_MARKET_TARGET_PAIRS}
 - SERVICE_MARKET_TARGET_PAIRS=${SERVICE_MARKET_TARGET_PAIRS}

**Review by nagdahimanshu — COMMENTED** [review]
Please cherry-pick the commit `bd232aabd59995745b636bcd1e27385817337715` to fix current eslint issues

File: README.md [hunk 14]
```diff
@@ -22,6 +22,7 @@ Each service is an independent part of the repository and is placed in a separat
 | [Lisk](services/core) | The Lisk Core service acts as a bridge between the Lisk Core and the Lisk Service API. Its main purpose is to provide enriched data from the Lisk Core API. This service is aimed at providing high availability, and both efficient and reliable access to the Lisk Core API. |
 | [Market](services/market) | The Market service allows price data retrieval. It supports multiple sources to keep the current Lisk token price up-to-date and available to the clients in real time. |
 | [Newsfeed](services/newsfeed) | The Newsfeed service is a single-purpose microservice that aggregates content sharing platforms and shares collected data with UI clients such as Lisk Desktop. |
+| [Export](services/export) | The Export service allows export of transactions stored in the blockchain. |
```
Comment (sameersubudhi) [inline README.md:25]:
```suggestion
| [Export](services/export) | The Export service enabled users to download the transaction history as a CSV file for any given account on the blockchain. |
```

File: docker-compose.yml [hunk 15]
```diff
@@ -177,15 +177,44 @@ services:
...
+      - SERVICE_MARKET_TARGET_PAIRS=${SERVICE_MARKET_TARGET_PAIRS}
```
Comment (sameersubudhi) [inline docker-compose.yml:192]:
```suggestion
```

**Review by sameersubudhi — APPROVED** [review]
Looks good. However, kindly consider the following changes.

File: README.md (hunk 14 above)
Comment (MichalTuleja) [reply README.md:25]: Thanks Sameer

File: docker-compose.yml (hunk 15 above)
Comment (MichalTuleja) [reply docker-compose.yml]: Updated

**Review by nagdahimanshu — APPROVED** [review]
Looks good to me 👍

**Pull request merged**